src/components/smart-screen/smart-screen.tsx

```diff
@@ -158,6 +158,7 @@ class SmartScreenComp extends React.Component<IReduxProps & IExternalProps, ISta
                     setScreenInputData: this.props.setScreenInputData
                 }}
                 blockchain={this.props.account.blockchain}
+                scrollPubSub={undefined}
```

Comment:
change this to pubSub, we could send here different kind of events not only scroll events